OpenTelemetry.sln

```diff
@@ -224,6 +224,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.Open
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApp.AspNetCore.6.0", "test\TestApp.AspNetCore.6.0\TestApp.AspNetCore.6.0.csproj", "{0076C657-564F-4787-9FFF-52D9D55166E8}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "learning-prometheus-http-server", "docs\metrics\learning-prometheus-http-server\learning-prometheus-http-server.csproj", "{9E9465F6-4A36-45D1-8A82-A86CBDA02ECE}"
```

Comment:
Maybe also change the file name since we don't use "Prometheus HTTP Server" anymore?

Reply:
Updated filename. Thanks!